src/bindings/overlay.js

```diff
@@ -34,7 +35,8 @@ export function overlayElement(element, translation) {
     } else {
       // start with an inert template element and move its children into
       // `element` but such that `element`'s own children are not replaced
-      const tmpl = element.ownerDocument.createElement('template');
+      const tmpl =
+        document.createElementNS('http://www.w3.org/1999/xhtml', 'template');
```

Comment:
IIRC, we used `element.ownerDocument` to make sure principals where set correctly and methods like `importNode` could work.  I also think this was related to Gaia, although I don't quite remember the details.